docs/pages/react/solana-wallets/signing-and-sending-solana.mdx

```diff
@@ -0,0 +1,277 @@
+import { useSolanaTransaction } from "@account-kit/react";
+import { PublicKey, SystemProgram } from "@solana/web3.js";
+
+function MyComponent() {
+  const {
+    sendTransaction,
+    isPending,
+    signer,
+    reset,
+    data: { hash: txHash = null } = {},
+  } = useSolanaTransaction({});
+
+  if (!signer) {
+    return <div>Loading alchemy signer...</div>;
+  }
+
+  return (
+    <div>
+      Solana Address {signer.address}
+      <button
+        onClick={() =>
+          sendTransaction({
+            instructions: [
+              SystemProgram.transfer({
+                fromPubkey: new PublicKey(signer.address),
+                toPubkey: new PublicKey("<ToAddress>"),
+                lamports: someValue, // ex: lamports: 50000000 - 0.05 SOL
+              }),
+              SystemProgram.transfer({
+                fromPubkey: new PublicKey(signer.address),
+                toPubkey: new PublicKey("<ToAddress>"),
+                lamports: someValue2,
+              }),
+            ],
+          })
+        }
+      >
+        Go make Transfer
+      </button>
+      {!!txHash && <button onClick={() => reset()}> Reset </button>}
+      {!!txHash && (
+        <a
+          href={`https://explorer.solana.com/tx/${txHash}?cluster=devnet`}
+          target="_blank"
+        >
+          Go To Tracker
+        </a>
+      )}
+    </div>
+  );
+}
+```
+
+## How does sendTransaction work?
+
+The `sendTransaction` method will construct and send a Solana transaction using the connection you provide. You can either construct a transaction using simple parameters or instructions for finer grained control.
+
+Using simple transfer parameters:
+
+```tsx
+sendTransaction({
+  transfer: {
+    // Amount in the atomic values of solana (Sol)
+    amount: 123,
+    toAddress: "<AddressToSendTo>",
+  },
+});
+```
+
+Using instructions:
+
+If you want finer grained control like transfer and nonceAdvance you can pass instructions by using the exposed [web3 SystemProgram](https://solana-foundation.github.io/solana-web3.js/classes/SystemProgram.html).
+
+```tsx
+sendTransaction({
+  instructions: [
+    SystemProgram.transfer({
+      fromPubkey: new PublicKey("<FromAddress>"),
+      toPubkey: new PublicKey("<ToAddress>"),
+      lamports: params.transfer.amount,
+    }),
+  ],
+});
+```
+
+# Sign messages with useSolanaSignMessage
+
+To sign a message, either a strin or a byte array, using your Solana wallet, simply use the `useSolanaSignMessage` hook.
+
+```tsx [sign-solana-message.tsx]
+import { useSolanaSignMessage } from "@account-kit/react";
+import { SolanaSigner } from "@account-kit/signer";
+
+function MyComponent() {
+  const {
+    signer,
+    data,
+    signMessage,
+    data: signedMessage,
+    reset,
+  } = useSolanaSignMessage({});
+
+  if (!signer) {
+    return <div>Loading signer...</div>;
+  }
+
+  return (
+    <div>
+      <button onClick={() => signMessage({ message: "Here is my message" })}>
+        {" "}
+        Sign{" "}
```

Comment:
Nit: if you want to add spaces at the start and end, it might read a little nicer to do it in one line as `{" Sign "}`, likewise below.